src/widgets/dialogs/UserInfoPopup.cpp

```diff
@@ -701,7 +701,18 @@ void UserInfoPopup::setData(const QString &name,
                             const ChannelPtr &contextChannel,
                             const ChannelPtr &openingChannel)
 {
-    this->userName_ = name;
+    const QString idPrefix = "#";
```

Comment:
This (most likely) doesn't do what you want - it will always create a new `QString`. You can use `const QStringView ID_PREFIX = u"#"` (this is constexpr).

Reply:
```suggestion
    const QStringView idPrefix = u"#";
```